### PR TITLE
Niccolox https

### PR DESCRIPTION
**Pull request by niccolox**
this change alters the /ssl structure to /https and deprecates SSL in favor of TLS

I'll refresh the page content and screens next

### CODE REVIEW

File: content/https/_index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "HTTPS and TLS"
+description: "HTTPS TLS SSL on the Section Edge platform"
+weight: 700
+aliases:
+  - /ssl/
```
Comment (pavelnikolov) [inline content/https/_index.md]: Does this alias need to be changed to `https`?

File: content/dns/tutorials/configuring-a-new-domain.md [hunk 2]
```diff
@@ -37,5 +37,5 @@ All domains added to the Section platform will generate a unique CNAME record. T
 
 ### Step 3 - SSL Certificate
 
-Follow the how to guide on [provisioning an SSL certificate]({{< relref "ssl/how-tos/create-a-free-tls-certificate-using-letsencrypt.md" >}}).
+Follow the how to guide on [provisioning an SSL certificate]({{< relref "https/how-tos/create-a-free-tls-certificate-using-letsencrypt.md" >}}).
```
Comment (pavelnikolov) [inline content/dns/tutorials/configuring-a-new-domain.md]: Do we need to leave old URLs for backwards compatibility in search engines?

Comment (pavelnikolov) [reply content/dns/tutorials/configuring-a-new-domain.md]: Maybe we need 301 redirects for old URLs

Comment (niccolox) [reply content/dns/tutorials/configuring-a-new-domain.md]: yes, redirects/aliases should be done

```
---
title: "HTTPS and TLS"
description: "HTTPS TLS SSL on the Section Edge platform"
weight: 700
aliases:
  - /ssl/
---
```

**Pull request merged**